### PR TITLE
[NUI] ListView for Wearable

### PR DESCRIPTION
**Pull request by neostom432**
### Description of Change ###
ListView for wearable.
This pull request is temporally for reviewing.

### CODE REVIEW

File: src/Tizen.NUI/src/public/BaseComponents/View.cs [hunk 1]
```diff
@@ -746,6 +746,14 @@ public Size2D CurrentSize
             }
         }
 
+        public Size CurrentSizeFloat
```
Comment (rabbitfor) [inline src/Tizen.NUI/src/public/BaseComponents/View.cs:749]: Please hide it with [Browsable.Never] before ACR.

**Comment by rabbitfor** [comment]
Can you add an example link to see how to use it?
I want to compare usability with other frameworks.

**Comment by neostom432** [comment]
> Can you add an example link to see how to use it?
> I want to compare usability with other frameworks.

Here is the sample.
https://github.com/dalihub/nui-demo/tree/ListViewSample

File: src/Tizen.NUI.Components/Controls/RecyclerList/LayoutManager.cs [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+        }
+
+
+        protected float mPrevScrollPosition = 0.0f;
+        protected int mPrevFirstDataIndex = 0;
+
+        public View Container{get;set;}
+        public Size ItemSize{get;set;} = new Size();
+        public Orientation LayoutOrientation{get;set;} = Orientation.Vertical;
+        public float StepSize{
```
Comment (rabbitfor) [inline src/Tizen.NUI.Components/Controls/RecyclerList/LayoutManager.cs:26]: I think it is better to unify brace format with other major NUI source code, such View.cs

```
public float StepSize
{
    get
    {
        return mStepSize;
    }
}
```

File: src/Tizen.NUI.Components/Controls/RecyclerList/LayoutManager.cs [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+            }
+        }
+        protected float mStepSize = 0.0f;
+
+        public virtual void Layout(float scrollPosition)
+        {
+           
+        }
+
+        public virtual List<ListItem> Recycle(float scrollPositio)
```
Comment (rabbitfor) [inline src/Tizen.NUI.Components/Controls/RecyclerList/LayoutManager.cs:39]: scrollPositio -> scrollPosition

**Pull request closed without merging**